Linkedin_Connections_Scrapper/ReadMe.md

```diff
@@ -0,0 +1,38 @@
+# LinkedIn Connections Scrapper
+
+It's a script built with the help of Selenium and Pandas to scrap LinkedIn connections list along with the skills of each connection if you want to. Using just a oneline command you can sitback and have a CSV file prepared for your cause.
+
+# Installation
+
+Make sure you have the following Python libraries:
+> pip3 install selenium pandas
+
+The rest should be present as core Python modules.
+Next thing is to place ChromeDriver.exe in the same directory of the script. You can download it from [here](https://sites.google.com/a/chromium.org/chromedriver/downloads)
+(Note: Download the one with the same version of your Chrome browser.)
+
+# Usage
```

Comment:
Explain the differences between basic and scrapping usage. I mean a one-liner for the expected output